@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getInput.sqf

```diff
@@ -29,7 +29,7 @@ private _isZeus             = isNull findDisplay 312;
 private _inMap              = !visibleMap;
 private _inInventory        = isNull findDisplay 602;
 
-private _isPlaying          = isGameFocused && _paused && _chatting && _inDialog && _isZeus && _inMap && _inInventory && !fza_ah64_lastFrameGetIn; 
+private _isPlaying          = !freeLook && isGameFocused && _paused && _chatting && _inDialog && _isZeus && _inMap && _inInventory && !fza_ah64_lastFrameGetIn; 
```

Comment:
The Freelook check needs to be linked to an addon option
(!freeLook && fza_ah64_disableFreelookInput)